roxctl/helm/derivelocalvalues/derivelocalvalues.go

```diff
@@ -2,14 +2,15 @@ package derivelocalvalues
 
 import (
 	"context"
-	"fmt"
+	"io"
 	"os"
 	"path/filepath"
 	"strings"
 
 	"github.com/hashicorp/go-multierror"
 	"github.com/pkg/errors"
 	"github.com/stackrox/rox/pkg/maputil"
+	common2 "github.com/stackrox/rox/roxctl/common"
```

Comment:
```suggestion
	roxctlcommon "github.com/stackrox/rox/roxctl/common"
```
or rename helm common